### PR TITLE
Update/new careers page

### PR DESCRIPTION
**Pull request by RodrigoCastroS**
Update the current careers page realigning the modules and change the order of appearance. There is also a new Locations module on the page that was developed on the new/job-listing page branch.

### CODE REVIEW

File: version_control/Codurance_September2020/css/pages/careers-v2.css [hunk 1]
```diff
@@ -86,6 +86,21 @@ body{
     }
 {% endcall %}
 
+/*------------------------------------*\
+  #Locations Section
+\*------------------------------------*/
+
+.locations{
+    background: linear-gradient(180deg,hsla(24.2,86.3%,54.3%,0.3) 0%,white 100%);
```
Comment (Dan-Bird) [inline version_control/Codurance_September2020/css/pages/careers-v2.css:94]: Are these one off colour values? If we require this value in other areas then we could add it to the design system.

File: version_control/Codurance_September2020/css/pages/careers-v2.css [hunk 2]
```diff
@@ -86,6 +86,21 @@ body{
...
+}
+
+.locations_container{
```
Comment (Dan-Bird) [inline version_control/Codurance_September2020/css/pages/careers-v2.css:97]: Should this be a double underscore or a hyphen?

File: version_control/Codurance_September2020/templates/careers-v2.html [hunk 3]
```diff
@@ -50,10 +50,25 @@ <h1>Careers</h1>
...
   </section>
+  <section class="craftspeople">
+    {% module 'craftspeople' path='../modules/Text-and-image.module' label="Craftspeople section" no_wrapper=True%}
+  </section>
+  <section class="testimonials">
+    {% module 'Testimonial Slider' path='../modules/Landing Page Modules/LP-testimonial-slider.module' label="Testimonial Slider"  no_wrapper=True %}
+  </section>
 
+  <section class="locations">
+    <div class="locations_container">
```
Comment (Dan-Bird) [inline version_control/Codurance_September2020/templates/careers-v2.html:61]: Similar to the CSS, should this be a double underscore or a hyphen?

**Review by Dan-Bird — APPROVED** [review]
👍

File: version_control/Codurance_September2020/css/pages/careers-v2.css (hunk 2 above)
Comment (RodrigoCastroS) [reply version_control/Codurance_September2020/css/pages/careers-v2.css:97]: double underscore. Thanks!

File: version_control/Codurance_September2020/templates/careers-v2.html (hunk 3 above)
Comment (RodrigoCastroS) [reply version_control/Codurance_September2020/templates/careers-v2.html:61]: double underscore. Thanks!

File: version_control/Codurance_September2020/css/pages/careers-v2.css (hunk 1 above)
Comment (RodrigoCastroS) [reply version_control/Codurance_September2020/css/pages/careers-v2.css:94]: It is... kind of. When designers are creating gradients, sometimes they use different directions and in this case, they might use the base colour (in this case persimmon) but with different hue and alpha values. So I don't know if I would be able to change the value of a colour property value coming from a variable.

**Pull request merged**